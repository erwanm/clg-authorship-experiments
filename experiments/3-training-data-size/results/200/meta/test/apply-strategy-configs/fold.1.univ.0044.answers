1886-hj-pc 1897-cdw-tpfwsw	0.634324
1852-hbs-utc 1896-hj-toh	0.502551
1896-espw-tsasa 1905-fmc-fm	0.348816
1883-fmc-ars 1911-hj-to	0.458627
1882-wta-aa 1898-fmc-ari-v2	0.5904
1877-hj-ta 1880-hj-ws	0.414703
1893-fmc-md 1905-fmc-wso	0.436665
1887-fmc-s 1891-fmc-k	0.711208
1885-wta-sts 1894-cdw-tgh	0.612362
1871-haj-tt 1883-haj-tbb	0.370778
1852-hbs-utc 1871-hbs-pawt	0.39274
1895-cfw-mcac 1906-mt-cfma	0.414703
1866-wta-bos 1876-wta-vao	0.436665
1864-lma-m 1904-rwc-sm	0.326854
1886-hj-tb 1900-espw-wtg	0.39274
1881-mt-tpatp 1900-espw-wtg	0.370778
1872-haj-ptf 1898-haj-abf	0.436665
1882-fmc-mi 1908-fmc-tws	0.886905
1898-hj-itc 1901-hj-tsf	0.370778
1871-espw-tsp 1907-espw-tldup	0.546475
1871-espw-tsp 1911-rwc-tcl	0.601397
1866-wta-ob 1883-haj-tbb	0.39274
1871-hbs-pawt 1880-hj-c	0.568438
1872-haj-ptf 1893-haj-dtn	0.436665
1896-mt-projoa 1907-ab-rm	0.458627
1888-es-ttatv 1893-fmc-kl-v2	0.348816
1897-ab-tdohy 1911-us-alp	0.623359
1885-fmc-zamc 1887-fmc-s	0.678248
1906-mt-cfma 1915-rwc-b	0.502551
1885-fmc-tcotk 1900-fmc-itpotk	0.458627
1883-fmc-dc 1890-fmc-acmr	0.480589
1906-fmc-alor 1908-fmc-tp	0.864943
1889-fmc-g 1893-fmc-md	0.623359
1857-wta-ta 1895-ab-mgtonel	0.348816
1883-hj-rh 1890-hj-ttm	0.326854
1883-fmc-tl 1886-hj-pc	0.524513
1895-fmc-t 1910-us-sts	0.217043
1897-cdw-tpfwsw 1922-rwc-tfj	0.524513
1898-hj-ttm 1917-ab-bn	0.326854
1864-lma-opdaot 1897-ewaoc-tdof	0.480589
1882-haj-ffbts 1886-hbs-tsw	0.5904
1895-wta-alae 1899-hj-taa	0.458627
1871-haj-tt 1878-hbs-pp	0.39274
1908-us-tm 1921-us-tbol	0.722172
1854-hbs-smofl-v2 1896-mt-projoa	0.436665
1881-hj-tpoal 1903-hj-ta	0.502551
1893-fmc-md 1906-mt-cfma	0.480589
1901-espw-a 1904-espw-t	0.348816
1882-wta-aa 1890-wta-otb	0.799056
1880-hj-c 1896-hj-toh	0.744135
1858-wta-pap 1870-wta-faf	0.39274
1891-fmc-do 1894-fmc-wg	0.546475
1868-lma-lw 1898-haj-abf	0.480589
1884-wta-Sac 1910-us-sts	0.370778
1881-hj-tpoal 1885-hj-gr	0.963789
1888-fmc-si 1894-fmc-wg	0.568438
1905-fmc-fm 1908-fmc-tws	0.436665
1866-haj-hf 1916-mt-tms	0.689246
1880-hj-ws 1885-hj-altof	0.436665
1914-us-sm 1921-us-tbol	0.502551
1903-us-tjoas 1908-us-tm	0.755132
1878-hj-te 1902-hj-twotd	0.546475
1866-haj-hf 1893-haj-ianwatgfoa	0.546475
1885-hj-altof 1888-hj-tap	0.436665
1895-fmc-ajs 1898-fmc-ari-v2	0.326854
1874-haj-rftr 1922-us-tcmc	0.326854
1869-wta-dtr 1910-ew-tomag	0.480589
1855-hbs-tmf 1883-fmc-tl	0.546475
1880-hj-ws 1894-espw-asl	0.436665
1864-lma-opdaot 1883-fmc-tl	0.502551
1858-wta-pap 1874-wta-tcw	0.711208
1880-hj-ws 1913-ew-tcotc	0.436665
1859-hbs-tmw 1862-hbs-tpooi	0.5904
1871-haj-tt 1891-haj-tetb	0.524513
1886-hj-tb 1904-hj-tgb	0.502551
1896-hj-tsop 1918-rwc-tlg	0.436665
1866-haj-hf 1898-haj-abf	0.480589
1867-espw-gyatgc 1900-espw-wtg	0.546475
1893-fmc-kl-v2 1893-fmc-tr	0.5904
1895-espw-tsojc 1898-fmc-vc	0.612362
1883-fmc-dc 1893-fmc-md	0.458627
1896-espw-cfal 1905-haj-fftf	0.821018
1898-hj-itc 1921-us-tbol	0.73317
1903-us-og 1918-us-jh	0.568438
1874-wta-tcw 1880-wta-ds	0.458627
1876-haj-btlb 1892-haj-dfg	0.348816
1878-hj-aie 1909-rwc-tdm	0.39274
1902-fmc-c 1916-ab-tp	0.458627
1883-hj-rh-v2 1897-hj-wmk	0.414703
1895-wta-alae 1898-us-apom	0.634324
1888-hj-tr 1893-haj-dtn	0.524513
1885-fmc-atoalp 1898-fmc-vc	0.524513
1885-fmc-atoalp 1905-fmc-fm	0.458627
1869-espw-mwag 1889-espw-tsfi	0.436665
1891-fmc-do 1891-wta-sbtu	0.458627
1887-fmc-s 1888-haj-bb	0.326854
1893-haj-ianwatgfoa 1894-cdw-tgh	0.326854
1868-espw-tga 1917-hj-tit	0.480589
1883-hj-rh-v2 1904-hj-tgb	0.502551
1898-hj-itc 1908-fmc-tp	0.502551
