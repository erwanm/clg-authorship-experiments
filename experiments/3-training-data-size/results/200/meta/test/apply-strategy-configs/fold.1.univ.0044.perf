0.374662	0.614200	0.610000
