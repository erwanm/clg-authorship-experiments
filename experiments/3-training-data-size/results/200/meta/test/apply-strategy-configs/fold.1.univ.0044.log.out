apply-config.sh: applying config '/tmp/expe3/200/process/selected-meta-configs/fold.1.0076.model/strategy-configs/fold.1.univ.0044' to cases '/home/moreaue/dump/expe3//results/200/meta/test/apply-strategy-configs/only-cases.tmp.0ie5E36C'; results written to '/home/moreaue/dump/expe3//results/200/meta/test/apply-strategy-configs/fold.1.univ.0044'
apply-config.sh: done.
